testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/preload/PreloadMiniAppWindow.kt

```diff
@@ -49,8 +49,8 @@ class PreloadMiniAppWindow(private val context: Context, private val preloadMini
 
         if (!isAccepted()) {
             launchScreen()
-        } else if (!doesDataExist()) {
-            storeAcceptance(DEFAULT_ACCEPTANCE) // should be false only after accept.
+        } else if (!prefs.contains(miniAppId)) {
```

Comment:
What is the difference of those statements`if (!isAccepted())` and `else if (!prefs.contains(miniAppId))`?

Reply:
Right, yeah, I'll change this part in 2505 ticket.